### DISCUSSION TITLE
Can I install PiVPN for OpenVPN if I already have OpenConnect Server (ocserv) installed in it?

### DISCUSSION THREAD

**Discussion by crxssrazr93** [pivpn/pivpn#1747] [Q&A]
I currently have Ocserv running on my VPS, and would like to know if I can install OpenVPN-AS as well?

Currently ocserv is using TCP 443.

I assume I'd need to have OpenVPN use a port other than 443, no?

Never used OpenVPN before, and new to VPNs in general, so I am a bit confused as to how it works.

**Comment by coolapso** [comment]
as long as OpenVPN does not run on the same port of ocserv you should be fine.
